### PR TITLE
Bump babel to version 6.14.0 - fixes class inheritance in IE <=10

### PR DESCRIPTION
**Pull request by joaovpmamede**
Fixes https://github.com/istarkov/google-map-react/issues/223 
Updated Babel packages to `6.14.0`. More info [here](https://github.com/babel/babel/pull/3527)

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -1,6 +1,4 @@
-src
```
Comment (istarkov) [inline .npmignore]: why? imo current configuration is good enough

**Comment by istarkov** [comment]
I'll merge this after you remove your changes to .npmignore
Sources are available on github, I see no reason to move them into `npm`

File: .npmignore (hunk 1 above)
Comment (joaovpmamede) [inline .npmignore]: Erm...sorry about this, I was testing something on my repo and by mistake I also pushed this.

**Comment by joaovpmamede** [comment]
Done and sorry for that.

**Comment by istarkov** [comment]
Thank you!!! Looks I forgot to made clean reinstall of `node_modules` last months, and this deps become a problem.

**Comment by istarkov** [comment]
Will publish when `npm install` be completed.

**Comment by istarkov** [comment]
0.18.3, I wish you never wrote code for IE-9-10 again ;-)

**Comment by joaovpmamede** [comment]
Hehe I wish that too...probably just in one or two years from now.

**Pull request merged**